web-wallet/src/lib/components/Send/Send.svelte

```diff
@@ -355,7 +379,13 @@
     <WizardStep step={3} {key} showNavigation={false}>
       <OperationResult
         errorMessage="Transaction failed"
-        operation={execute(sendToAddress, sendAmountInLux, gasPrice, gasLimit)}
+        operation={execute(
+          sendToAddress,
+          sendAmountInLux,
+          gasPrice,
+          gasLimit,
+          memo
```

Comment:
Nitpicking: I'd leave the gas arguments as last for consistency with other methods.